.github/workflows/integration_test.yml

```diff
@@ -1,9 +1,10 @@
 name: Azure integration test
 
 on:
+  workflow_dispatch:
   schedule:
     - cron: "0 22 * * 0" # every sunday at 10pm
-  push:
+  pull_request:
```

Comment:
Yes it was intended, I thought it would actually make more sense, instead of running it after merging "on push".

It will also be triggered again when we create a release as part of the release workflow. What do you think?